DataUsageAgreement.md

```diff
@@ -0,0 +1,27 @@
+Texas Justice Initiative     
+P.O. Box 164286  
+Austin, TX 78716   
+https://www.texasjusticeinitiative.org  
+
+![](https://raw.githubusercontent.com/texas-justice-initiative/website-nextjs/master/images/tji-logo.svg) 
+
+# Data Access License Terms
+
+This agreement ("Agreement") is entered into between the Texas Justice Initiative ("TJI") and any individual(s) (a "Recipient") who downloads or accesses any data from TJI’s website (the "Datasets" as described more fully herein).
+
+TJI developed and owns the Datasets on its [website](https://texasjusticeinitiative.org/data), which represent a collection and curation of data and information gathered from various governmental agencies related to Texas criminal justice issues. TJI is willing to grant Recipient access to the Dataset on the following terms:
+
+## 1. Limited License
+By downloading or accessing any Datasets, you hereby agree to all of the terms specified in this license. TJI grants  Recipient the limited right to use the Datasets only for the Recipient’s own academic, nonprofit, and research purposes. The Recipient may not use TJI’s Datasets for any commercial or profit-making purpose; and may only use the Dataset in accordance with this Agreement. TJI retains all rights, title, and interest in and to the Dataset, and does not grant any rights in the Dataset other than as specified herein.
+
+## 2. Attribution 
+When using TJI Datasets in any reports, articles, papers, journals or other mediums you must document and give TJI (appropriate credit)[https://creativecommons.org/licenses/by/3.0/us/#], provide a link to TJI’s website, and specifically indicate what, if any, changes were made to the Datasets. You may do so in any reasonable manner, but you may not in any way  suggest that TJI endorses you or your use of its Datasets.
```

Comment:
```suggestion
When using TJI Datasets in any reports, articles, papers, journals or other mediums you must document and give TJI [appropriate credit](https://creativecommons.org/licenses/by/3.0/us/#), provide a link to TJI’s website, and specifically indicate what, if any, changes were made to the Datasets. You may do so in any reasonable manner, but you may not in any way  suggest that TJI endorses you or your use of its Datasets.
```

I swap the parentheses and square brackets in Markdown links by accident every day 😆

Reply:
thanks for catching this!!!